Kommunicate/Classes/KMConversationService.swift

```diff
@@ -92,6 +92,12 @@ public class KMConversationService: KMConservationServiceable {
         }
     }
 
+    public func statusForUser(_ groupId: NSNumber, completion: @escaping (String?) -> ()) {
+        awayMessageFor(applicationKey: ALUserDefaultsHandler.getApplicationKey(), groupId: groupId, completion: { message in
+            completion(message)
+        })
+    }
```

Comment:
Can we directly use awayMessageFor method outside? This method is doing nothing except for calling another method.